Content.Server/SqueezeBottle/SqueezeBottleSystem.cs

```diff
@@ -0,0 +1,102 @@
+using Content.Server.Administration.Logs;
+using Content.Server.Chemistry.Containers.EntitySystems;
+using Content.Shared.Database;
+using Content.Shared.IdentityManagement;
+using Content.Shared.Interaction;
+using Content.Shared.Item;
+using Content.Shared.SqueezeBottle;
+using Content.Shared.Nutrition.EntitySystems;
+using Content.Shared.Popups;
+using Content.Shared.Verbs;
+using Robust.Shared.Audio.Systems;
+
+using Content.Server.ReagentOnItem;
+
+namespace Content.Server.SqueezeBottle;
+
+/// <summary>
+///     Squeeze bottles are to apply reagents to items. For example,
+///     a glue bottle is a squeeze bottle because you need to be
+///     able to apply the glue to items!
+/// </summary>
+public sealed class SqueezeBottleSystem : EntitySystem
```

Comment:
You can probably rename this something more generic.

Reply:
What about "ApplyReagentToItem"? I guess I didn't think about other items using it but it makes sense 😆

Reply:
That probably works.